### PR TITLE
Svg_parser: cleanups and some outline-related fixes

### PR DESCRIPTION
**Pull request by rodolforg**
- Outline width
- Fill x Outline z-order (paint-order)

### CODE REVIEW

File: synfig-core/src/modules/mod_svg/svg_parser.cpp [hunk 1]
```diff
@@ -273,180 +273,163 @@ void
...
-					build_fill(child_stroke,stroke,mtx);
-				else
-					build_fill(child_stroke,stroke,SVGMatrix::identity);
-			}	
+				build_fill(child_stroke, stroke, bline_matrix);
+			}
+		};
+
+		bool fill_under_stroke = true;
+		if (typeStroke != FILL_TYPE_NONE && typeStroke != FILL_TYPE_NONE) {
```
Comment (ice0) [inline synfig-core/src/modules/mod_svg/svg_parser.cpp:413]: The condition on the left and right sides is the same.

File: synfig-core/src/modules/mod_svg/svg_parser.cpp [hunk 2]
```diff
@@ -456,6 +439,86 @@ Svg_parser::parser_graphics(const xmlpp::Node* node, xmlpp::Element* root, Style
 	}
 }
 
+
+void
+Svg_parser::build_region(xmlpp::Node* root, Style style, const std::list<BLine>& k, const String& desc)
+{
+	String fill			    = style.get("fill", "none");
```
Comment (ice0) [inline synfig-core/src/modules/mod_svg/svg_parser.cpp:446]: Indent.

File: synfig-core/src/modules/mod_svg/svg_parser.cpp [hunk 3]
```diff
@@ -456,6 +439,86 @@ Svg_parser::parser_graphics(const xmlpp::Node* node, xmlpp::Element* root, Style
...
+	String fill_rule		= style.get("fill-rule", "evenodd");
+	String stroke			= style.get("stroke", "none");
```
Comment (ice0) [inline synfig-core/src/modules/mod_svg/svg_parser.cpp:448]: `stroke` is not used.

Comment (rodolforg) [reply synfig-core/src/modules/mod_svg/svg_parser.cpp]: Done

File: synfig-core/src/modules/mod_svg/svg_parser.cpp (hunk 2 above)
Comment (rodolforg) [reply synfig-core/src/modules/mod_svg/svg_parser.cpp]: Done

File: synfig-core/src/modules/mod_svg/svg_parser.cpp (hunk 1 above)
Comment (rodolforg) [reply synfig-core/src/modules/mod_svg/svg_parser.cpp]: Done

**Comment by rodolforg** [comment]
Rebased

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**